test/verify.sh

```diff
@@ -239,6 +262,14 @@ unzip -q /tmp/wlp.zip -d /opt/ibm
    fi
 }
 
+testDockerOnOpenShift()
+{
+   testLibertyStarts "OpenShift"
+   testLibertyStops "OpenShift"
+   testLibertyStopsAndRestarts "OpenShift"
+   testFeatureList "OpenShift"
```

Comment:
@Rachelwrq I did some testing and `testFeatureList` is where we are spending a bunch of time because of the large file download.  Can you try removing this line and see if the build is able to finish in under 2 hours?  We shouldn't need to run this test twice for each image anyway.  

If it still hits the max time without this call, we might have to remove testFeatureList entirely.